doc / ongoing v0.37 (#283)
CoinAlpha/hummingbot-docs: content/release-notes/0.37.0.mdx

Read more about how to use the ProBit connector [here](/exchange-connectors/probit).

## New Exchange Connector: Beaxy

We are glad to present to you that Hummingbot now supports [Beaxy](https://beaxy.com/) exchange! Beaxy is a digital asset exchange that aims to set itself apart from others in the field by offering a feature rich platform built from the ground up with clients of every experience level in mind.

Beaxy is compatible with [pure market making](https://docs.hummingbot.io/strategies/pure-market-making/), [cross-exchange market-making](/strategies/cross-exchange-market-making/), and [arbitrage strategies](/strategies/arbitrage/).

Read more about how to use the Beaxy connector [here](/exchange-connectors/Beaxy).

## New Protocol Connector: Perpetual Finance

In v0.37, Hummingbot has added a new protocol connector, [Perpetual Finance](https://perp.fi/).

Perpetual protocol is a decentralized protocol to trade perpetual contracts for every asset powered by a Virtual Automated Market Maker (vAMM). The project aims to create novel financial instruments by democratizing Futures and other crypto-asset derivatives.

> Read more about how to use the Perpetual Finance connector [here](/protocol-connectors/perp-fi)

Reviewer: ```suggestion
Read more about how to use the Perpetual Finance connector [here](/protocol-connectors/perp-fi).
```

For consistency.

## Introducing the new Spot Perpetual Arbitrage strategy (Beta)

In v0.37, we introduce the new Spot Perpetual Arbitrage strategy! This strategy lets you arbitrage between a spot and futures exchange. The differences between the prices on the spot and futures markets before the settlement date give a good sense of the direction in which both markets would go. Prices on both futures and spot will converge as the settlement date or time approaches.

> The idea of this strategy is to open a position in one direction and execute an order in the opposite direction on the spot. Learn more about it here #[2977](https://github.com/CoinAlpha/hummingbot/pull/2977).

Reviewer: "~~Learn more about it here #[2977](https://github.com/CoinAlpha/hummingbot/pull/2977)~~" we should avoid directing users to PRs and direct them instead to the section in the docs where these changes are explained in more detail.

> ## Improvement to Liquidity Mining strategy

Reviewer: "Improvements" since there are more than one.

> In v0.36, we introduced the Liquidity Mining strategy. In v0.37, we made improvements to give users a better experience when using this strategy, we added two new parameters [`max_spread`](/strategies/liquidity-mining/#max_spread) and [`max_order_age`](/strategies/liquidity-mining/#max_order_age). Tables are also added to let you see more detail about your base and quote balance and also the campaign. Another improvement is better balance distribution on the selected `token`. Learn more about it here #[2961](https://github.com/CoinAlpha/hummingbot/pull/2961).

Reviewer: ```suggestion
We added two new parameters [`max_spread`](/strategies/liquidity-mining/#max_spread) and [`max_order_age`](/strategies/liquidity-mining/#max_order_age) to Liquidity Mining strategy to give users a more functionality when using the strategy.

Tables are added to display the campaign, base and quote balance in more detail. Balance distribution has also been improved on the user's selected `token`.

Learn more about it here #[2961](https://github.com/CoinAlpha/hummingbot/pull/2961).
```

Reviewer: "~~Learn more about it here #[2961](https://github.com/CoinAlpha/hummingbot/pull/2961)~~" ditto

Reviewer: @JeremyKono this is not resolved 👆

Author: Resolved in the latest commit

## Bug Fixes

- #[2949](https://github.com/CoinAlpha/hummingbot/pull/2949) Add `Exchanges Total` to `balance` command

>   - **We want to thank community member [Iamtha](https://github.com/lamtha) for these contribution!**

Reviewer: ```suggestion
  - **We want to thank community member [Iamtha](https://github.com/lamtha) for this contribution!**
```
